questionnaire/questionnaireUISchema.js

```diff
@@ -523,5 +560,54 @@ module.exports = {
             buttonText: 'Agree and submit',
             pageContext: 'submission'
         }
+    },
+    'p-applicant-select-treatments': {
+        // transformer: 'form',
+        options: {
+            transformOrder: [
+                'q-applicant-other-treatment-dmi',
+                'q-applicant-select-treatments-dmi'
+            ],
+            outputOrder: ['q-applicant-select-treatments-dmi'],
+            properties: {
+                'q-applicant-select-treatments-dmi': {
+                    options: {
+                        conditionalComponentMap: [
+                            {
+                                itemValue: 'other',
+                                componentIds: ['q-applicant-other-treatment-dmi']
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    },
+    'p-gp-enter-your-address': {
+        options: {
+            properties: {
+                'q-gp-building-and-street': {
+                    options: {
+                        macroOptions: {
+                            classes: ''
```

Comment:
is this to get rid of the default widths applied to them?

Reply:
Yes. This is the same as the page for the applicants address, the width of the top to lines doesn't match the size derived from the character count, but this was requested for aesthetic reasons.